pkg/tfbridge/provider_test.go

```diff
@@ -525,13 +540,13 @@ func TestProviderCheckV2(t *testing.T) {
 	failures := testCheckFailures(t, provider, "SecondResource")
 	sort.SliceStable(failures, func(i, j int) bool { return failures[i].Reason < failures[j].Reason })
 	assert.Equal(t, "Conflicting configuration arguments: \"conflicting_property\": conflicts with "+
-		"conflicting_property2. Examine values at 'SecondResource.ConflictingProperty'.", failures[0].Reason)
+		"conflicting_property2. Examine values at 'name.conflictingProperty'.", failures[0].Reason)
 	assert.Equal(t, "", failures[0].Property)
 	assert.Equal(t, "Conflicting configuration arguments: \"conflicting_property2\": conflicts with "+
-		"conflicting_property. Examine values at 'SecondResource.ConflictingProperty2'.", failures[1].Reason)
+		"conflicting_property. Examine values at 'name.conflictingProperty2'.", failures[1].Reason)
```

Comment:
As a result of unifying PF and SDKv2 versions of the code, it now uses resource name from the URN "name" which is more informative normally than the type name (SecondResource).